dev-python/sympy/files/sympy-1.11-no-deprecated-threading-event-api.patch

```diff
@@ -0,0 +1,51 @@
+commit b89ca436cf25e2a1caf82c9d33221d06698d0ff5
```

Comment:
Can you include a link to the patch source at the top? Thanks

Reply:
OK